test/lock-parser/fixtures/additional-global-options/out.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Would like to also keep all the meta from this file, imagine we want to in the future parse the entire file and then get different sources to process is like a nuget specific processing then some other `muget` specific processing, but also we want to show as much meta s possible about this to the user.

We may even want to know about any private origins, so please grab every single piece of info available and add it as meta at the top as well.

Reply:
Not a blocker for now, but if easy please do add some stuff at top level as meta

Reply:
Will be adding the necessary information into meta as mentioned in comment on logic PR: https://github.com/snyk/snyk-paket-parser/pull/16/files#r255166038 and then regenerate the fixtures.